src-docs/src/components/guide_page/guide_page_header.tsx

```diff
@@ -62,6 +62,25 @@ export const GuidePageHeader: React.FunctionComponent<{}> = () => {
     );
   }
 
+  function renderFigma() {
+    const href = 'https://www.figma.com/community/file/1240039515407642648';
+    const label = 'OUI Figma component library';
+    return isMobileSize ? (
+      <OuiButtonEmpty size="s" flush="both" iconType="logoGithub" href={href}>
+        {label}
+      </OuiButtonEmpty>
```

Comment:
```suggestion
      <OuiButtonEmpty size="s" flush="both" iconType="logoFigma" href={href}>
        {label}
      </OuiButtonEmpty>
```

Should this be `logoFigma`?